src/settings/ProfileSettingsTab.tsx

```diff
@@ -78,64 +73,51 @@ export function ProfileModal({ client, ...rest }: Props) {
     [saveProfile, removeAvatar]
   );
 
-  useEffect(() => {
-    if (success) {
-      onClose();
-    }
-  }, [success, onClose]);
-
   return (
-    <Modal title={t("Profile")} isDismissable {...rest}>
-      <ModalContent>
-        <form onSubmit={onSubmit}>
-          <FieldRow className={styles.avatarFieldRow}>
-            <AvatarInputField
-              id="avatar"
-              name="avatar"
-              label={t("Avatar")}
-              avatarUrl={avatarUrl}
-              displayName={displayName}
-              onRemoveAvatar={onRemoveAvatar}
-            />
-          </FieldRow>
-          <FieldRow>
-            <InputField
-              id="userId"
-              name="userId"
-              label={t("User ID")}
-              type="text"
-              disabled
-              value={client.getUserId()}
-            />
-          </FieldRow>
-          <FieldRow>
-            <InputField
-              id="displayName"
-              name="displayName"
-              label={t("Display name")}
-              type="text"
-              required
-              autoComplete="off"
-              placeholder={t("Display name")}
-              value={displayName}
-              onChange={onChangeDisplayName}
-            />
-          </FieldRow>
-          {error && (
-            <FieldRow>
-              <ErrorMessage error={error} />
-            </FieldRow>
-          )}
-          <FieldRow rightAlign>
-            <Button type="button" variant="secondary" onPress={onClose}>
-              Cancel
-            </Button>
-            <Button type="submit" disabled={loading}>
-              {loading ? t("Saving…") : t("Save")}
-            </Button>
-          </FieldRow>
-        </form>
-      </ModalContent>
-    </Modal>
+    <form onSubmit={onSubmit}>
+      <FieldRow className={styles.avatarFieldRow}>
+        <AvatarInputField
+          id="avatar"
+          name="avatar"
+          label={t("Avatar")}
+          avatarUrl={avatarUrl}
+          displayName={displayName}
+          onRemoveAvatar={onRemoveAvatar}
+        />
+      </FieldRow>
+      <FieldRow>
+        <InputField
+          id="userId"
+          name="userId"
+          label={t("User ID")}
```

Comment:
```suggestion
          label={t("Username")}
```

This is what the MXID is called on the login and registration pages, so we should be consistent with it